extensions/data-transfer/portability-data-transfer-google/src/main/java/org/dataportabilityproject/datatransfer/google/calendar/GoogleCalendarExporter.java

```diff
@@ -81,13 +85,39 @@ private static CalendarAttendeeModel transformToModelAttendee(EventAttendee atte
     return new CalendarEventModel.CalendarEventTime(offsetDateTime, dateTime.getDate() != null);
   }
 
+  private static RecurrenceRule getRecurrenceRule(List<String> ruleStrings) {
+    RecurrenceRule.Builder ruleBuilder = new RecurrenceRule.Builder();
+    for (String st : ruleStrings) {
+      String[] split = st.split(":");
+      String type = split[0];
```

Comment:
Let's add a precondition to ensure there are two parts or meets a certain format requirement